packages/openssl/build

```diff
@@ -9,9 +9,7 @@ pushd "/pkg/src/openssl"
   --openssldir="$PKG_PATH/etc/ssl" \
   shared \
   linux-x86_64 \
-  no-krb5 \
   no-rc4 \
-  no-ssl2 \
```

Comment:
Why are we changing OpenSSL build instructions without changing the OpenSSL version?

Reply:
@jongiddy sorry, the answer is hidden in commit messages. This has been a noop for a while, and it will be an error with OpenSSL 1.1.1.